scheduling-manager/src/cucumber/resources/features/scheduling/replace_after_terminate.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Deploying a Daemon environment to a cluster with Replace After Terminate
+
+  Background:
+    Given a Daemon environment named "DaemonEnvironment":
+      | clusterArn        | TestCluster           |
```

Comment:
I think the values go below the title not next to

Reply:
Cucumber places no restriction on how you parse the DataTable; the underlying test code can parse these values however it makes sense.

When we're specifying a large number of attributes on a small number (i.e. 1) of objects, it's more readable to have the table transposed like this as a large number of rows, rather than having a large number of columns.

Reply:
(But, if it's confusing to see both styles in the same spec, it's easy to change, just let me know)

Reply:
oh I see. Yea it's a little confusing to have it both ways